test/clj/centrebull/test/mock_generators.clj

```diff
@@ -19,6 +19,11 @@
          :shooter/preferred-name (last-name)
          :shooter/club           (string)}))
 
+(defn gen-range
+  ([] (gen-range (rand-int 99999)))
```

Comment:
This should be generating a uuid. Have a look at the competition generator (may require merging develop into this branch to get the function)